android/build-commissioner-libs.sh

```diff
@@ -67,6 +67,15 @@ cd ../
 
 rm -rf "$BUILD_DIR"/libs && mkdir -p "$BUILD_DIR"/libs
 
+## Check JNI contains any raw swig wrapper files
+
+if find "$BUILD_DIR"/src/java/io/openthread/commissioner -name "SWIGTYPE_p_std__*" -print -quit | grep -q .; then
```

Comment:
I think it's better to match all "SWIGTYPE_"
```suggestion
if find "$BUILD_DIR"/src/java/io/openthread/commissioner -name "SWIGTYPE_*" -print -quit | grep -q .; then
```